android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsViewModel.kt

```diff
@@ -26,6 +26,15 @@ class SurveyQuestionsViewModel(
     val questionContentUiModels: StateFlow<List<QuestionContentUiModel>> =
         _questionContentUiModels.asStateFlow()
 
+    private val _isSubmitSuccess = MutableStateFlow(false)
+    val isSubmitSuccess: StateFlow<Boolean> = _isSubmitSuccess.asStateFlow()
+
+    private val _isSubmitting = MutableStateFlow(false)
+    val isSubmitting: StateFlow<Boolean> = _isSubmitting.asStateFlow()
```

Comment:
Should we define submitting state instead separate into 2 different state? 🤔 

Ex:
```
SubmittingAnswerState {
    Submitting
    Success
     ....
}
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/123/commits/dfd30f14a98fe541a6cf2e25328309b3a9cacbc9